src/components/card/Card.css

```diff
@@ -0,0 +1,45 @@
+@font-face {
+  font-family: 'OldEnglish';
+  src: local('oldenglishtextmt'), url(./fonts/oldenglishtextmt.ttf) format('truetype');
+} 
+
```

Comment:
Card component should support responsive design.

![image](https://user-images.githubusercontent.com/48954743/226584816-96c4515f-9695-4af1-96e7-6b4cf248ae1c.png)